http/swagger.yml

```diff
@@ -9271,6 +9277,9 @@ components:
           type: string
         ySuffix:
           type: string
+        hoverDimension:
+          type: string
+          enum: [auto, x, y, xy]
```

Comment:
Should we mark one of these as the `default`?

Reply:
No. Empty string will be interpreted correctly by the UI.